config/config.go

```diff
@@ -94,7 +94,7 @@ type Config struct {
 
 		AccountSID string
 		AuthToken  string `password:"true" info:"The primary Auth Token for Twilio. Must be primary (not secondary) for request valiation."`
-		FromNumber string `public:"true" info:"The Twilio number to use for outgoing notifications."`
+		FromNumber string `displayName:"From Number or Messenger SID" public:"true" info:"The Twilio number or Messenger SID to use for outgoing notifications."`
```

Comment:
I think we need to tweak all references:

`Messenger SID` -> `Twilio Messaging Service SID`